### PR TITLE
Feature new theme

### PR DESCRIPTION
**Pull request by AliceGdj**
image

### CODE REVIEW

**Comment by jarjan** [comment]
[Create custom bootstrap theme](https://trello.com/c/DfJpndRM/68-create-custom-bootstrap-with)

**Comment by pheeria** [comment]
@AliceGdj @jarjan why did you return "Die Lernwekstatt" name next to the logo? :)

**Comment by jarjan** [comment]
> @AliceGdj @jarjan why did you return "Die Lernwekstatt" name next to the logo? :)

I did it, for SEO purposes, name of website should appear first in HTML :) 
But actually, I can hide it for browsers, but search engine bots will see that text still.

File: src/views/partials/main/header.hbs [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
-            <input type="hidden" value="https://lernwerkstatt-site.azurewebsites.net/" name="sites">
-            <input type="hidden" value="1" name="kh">
-            <input type="hidden" value="1" name="kn">
-            <input type="hidden" value="1" name="kac">
-            <span class="input-group-btn"><button class="btn btn-secondary my-2 my-sm-0" type="submit">
-                <i class="fa fa-search" aria-hidden="true"></i></button>
-            </span>
+  <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-primary">
+    <div class="container">
+      <a class="navbar-brand" href="/"><img height="24" src="/img/navbar_logo.png">Die Lernwerkstatt</a>
```
Comment (jarjan) [inline src/views/partials/main/header.hbs]: Added `Die Lerwerkstatt` on first header link for SEO purposes.

Comment (pheeria) [reply src/views/partials/main/header.hbs]: Can we hide it? It looked so clean without it :)

**Pull request merged**